esp/platform/espcontext.cpp

```diff
@@ -172,7 +173,7 @@ class CEspContext : public CInterface, implements IEspContext
     }
     virtual const MemoryBuffer & querySessionToken()
     {
-        return m_user->credentials().getSessionToken();
+        return m_user ? m_user->credentials().getSessionToken() : m_emptyMB;
```

Comment:
I would consider changing this function to return a "const char *" rather than MemoryBuffer and return a nullptr if the m_user was null. The only reasons, I can think of, for returning a MemoryBuffer would be if the caller needed to manipulate the buffer or if there was binary data in the buffer.  Wouldn't a return value of const char * be sufficient?